views/home.hbs

```diff
@@ -13,9 +13,10 @@
       Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec volutpat hendrerit !nibus.
     </p>
     <br>
-    <form action="/scan" method="post">
+    <form action="/scan" class="email-scan" method="post">
```

Comment:
Per #144, are we even using this file for anything? I was struggling to find a reference to this template.